meeteval/wer/api.py

```diff
@@ -0,0 +1,179 @@
+import glob
+import logging
+import re
+from pathlib import Path
+
+import meeteval.io
+from meeteval.wer.wer import ErrorRate
+
+__all__ = [
+    'cpwer',
+    'orcwer',
+    'mimower',
+    'tcpwer',
+]
+
+
+def _load_texts(
+        reference_paths: 'list[str]',
+        hypothesis_paths: 'list[str]',
+        regex,
+        reference_sort: 'bool | str' = False,
+        hypothesis_sort: 'bool | str' = False,
+        file_format=None,
+) -> 'tuple[meeteval.io.SegLST, meeteval.io.SegLST]':
+    """Load and validate reference and hypothesis texts.
+
+    Validation checks that reference and hypothesis have the same example IDs.
+    """
+
+    # Normalize and glob (for backwards compatibility) the path input
+    def _glob(pathname):
```

Comment:
Do we keep the globbing? I think that this can cause weird behavior. Especially because `_save_results` does not perform globbing currently

Reply:
I thought it is not difficult to keep it, so keeping means to not introduce a breaking change.
In `_save_results` it shouldn't be an issue. It can still create a reasonable name.

Reply:
Okay, the name just might be unexpected and contain `*`